Ansible/roles/cloudstack-manager/tasks/ubuntu.yml

```diff
@@ -55,21 +55,13 @@
   apt:
     update_cache: yes
 
-- name: Ensure CloudStack packages are installed
+- name: Ensure CloudStack management-server only is installed (we install usage server later)
   apt:
     pkg: "{{ item }}"
     state: present
     allow_unauthenticated: yes
   with_items:
     - cloudstack-management
-    - cloudstack-usage
-
-- name: remove CloudStack repository into sources list.
```

Comment:
I would not be happy about removing this step without understanding why it was there in the first place.
Making the occasional manual upgrade faster isn't as important as ensuring that we are creating representative environments.

Reply:
That's why I asked for your review - if it may break something (but we do leave repo on CentOS), then I will remove it?